src/TrackerCouncil.Smz3.Abstractions/TrackerCouncil.Smz3.Abstractions.csproj

```diff
@@ -11,4 +11,12 @@
       <ProjectReference Include="..\TrackerCouncil.Smz3.Shared\TrackerCouncil.Smz3.Shared.csproj" />
     </ItemGroup>
 
+    <ItemGroup>
+      <PackageReference Include="PySpeechServiceClient" Version="0.1.0" />
```

Comment:
If this is only available on Linux, maybe we should have a compiler symbol for this (if one isn't built-in already) and surrounding the code where it's used. Then we can avoid adding the dependency itself entirely on Windows if it'll never be used there anyway.

Reply:
Well, it is used for the grammar construction even on Windows, but I could possibly pull that out into a separate project/repo so that just the actual services themselves can be done that way. 

That being said, I do plan on making a Windows version just for possible testing for pronunciations if desired for Pink's streams. It's just a lower priority.

Reply:
I split it out so that grammar building is in its on nuget package so that it could be used on Windows whereas the actual services be Linux only, and I marked all of the functions as Linux only for now until I get it built for Mac and/or Windows. However, I was having issues getting OS dependent compiler constants working for actually being able to make the built code different. Might give it another go.